ROMFS/px4fmu_common/init.d/airframes/4061_atl_mantis_edu

```diff
@@ -142,57 +146,31 @@ param set-default MPC_Z_VEL_P 0.27
 
 # gimbal configuration
 param set-default MNT_MODE_IN 1
-param set-default MNT_MODE_OUT 2
-param set-default MNT_MAN_PITCH 1
-
+param set-default MNT_MODE_OUT 1
+param set-default MNT_MAN_PITCH 2
+param set-default MNT_RC_IN_MODE 1
+param set-default MNT_RATE_PITCH 0.5
 
 # RC
 param set-default RC_CHAN_CNT       12
 
 param set-default RC_MAP_THROTTLE   1
-param set-default RC_MAP_ROLL       2
+param set-default RC_MAP_YAW        2
 param set-default RC_MAP_PITCH      3
-param set-default RC_MAP_YAW        4
-param set-default RC_MAP_FLTMODE    5
-param set-default RC_MAP_AUX1       7
-
-param set-default RC1_DZ            10
-param set-default RC1_MAX           3413
-param set-default RC1_MIN           683
-param set-default RC1_REV           1
-param set-default RC1_TRIM          683
-param set-default RC2_DZ            10
-param set-default RC2_MAX           3413
-param set-default RC2_MIN           683
-param set-default RC2_REV           -1
-param set-default RC2_TRIM          2048
-param set-default RC3_DZ            10
-param set-default RC3_MAX           3413
-param set-default RC3_MIN           683
-param set-default RC3_REV           1
-param set-default RC3_TRIM          2048
-param set-default RC4_DZ            10
-param set-default RC4_MAX           3413
-param set-default RC4_MIN           683
-param set-default RC4_REV           -1
-param set-default RC4_TRIM          2048
-param set-default RC5_DZ            10
-param set-default RC5_MAX           3414
-param set-default RC5_MIN           2048
-param set-default RC5_REV           1
-param set-default RC5_TRIM          2048
-param set-default RC7_DZ            10
-param set-default RC7_MAX           3413
-param set-default RC7_MIN           683
-param set-default RC7_REV           1
-param set-default RC7_TRIM          2048
+param set-default RC_MAP_ROLL       4
+param set-default RC_MAP_AUX2       5
+param set-default RC_MAP_AUX3       10
+param set-default RC_MAP_AUX4       8
+param set-default RC_MAP_FLTMODE    6
+param set-default RC1_TRIM          1000
 
 
 # optical flow
 param set-default SENS_FLOW_MAXR   7.4
 param set-default SENS_FLOW_MINHGT 0.15
 param set-default SENS_FLOW_MAXHGT 5.0
-param set-default SENS_FLOW_ROT    4
-
+param set-default SENS_FLOW_ROT    0
 
-mixer load /dev/tap_esc /etc/mixers/quad_x.main.mix
+# ignore the SD card errors and use normal startup sound, and make it less loud
+set STARTUP_TUNE "1"
```

Comment:
We should ultimately have a way to do all of this from parameters or other board override mechanisms.

I'd like to get to the point where an airframe is nothing more than a declarative param file.

Reply:
Absolutely but probably not part of this PR.